ExilenceNextApp/src/components/settings-tabs/text-input-setting/TextInputSetting.styles.ts

```diff
@@ -0,0 +1,15 @@
+import { createStyles, makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles(theme =>
+  createStyles({
+    root: {
```

Comment:
I think we can skip `createStyles({ root: {} })` part and just go with
```
const useStyles = makeStyles(((theme: Theme) => ({
   marginTop: ...
})
```
Though @viktorgullmark could confirm that.

Reply:
Yup, I suppose so. I never quiet grasped the need for createStyles, probably related to typings? @sbsrnt

Reply:
Actually good catch. Should probably check documentation before talking gibberish :D
![image](https://user-images.githubusercontent.com/19862545/96705486-adb04f80-1395-11eb-8557-30c6431d9f96.png)

source: https://material-ui.com/guides/typescript/#using-createstyles-to-defeat-type-widening